app/utils/prepareAnswer.js

```diff
@@ -126,18 +126,22 @@ function prepareDailyStatsAnswer(body, n) {
 
 function prepareTestingResourceAnswer(stateResources) {
   if (stateResources.length === 0) {
-    return '\n\n\u{1F9EA} No COVID-19 Testing Labs found';
+    const temp = ['\n\n\u{1F9EA} No COVID-19 Testing Labs found'];
+    return temp;
   }
 
-  let data = `\n\n\u{1F9EA} COVID-19 Testing Labs in ${stateResources[0].state}, found ${stateResources.length} results\n`;
-
-  stateResources.forEach((stateResource) => {
-    data += `\n<b>${stateResource.nameoftheorganisation}, ${stateResource.city}\n`
-      + `</b>Description: ${stateResource.descriptionandorserviceprovided}.\n`
-      + `<i>Phone Number: ${stateResource.phonenumber}</i>\n`
-      + `<i>Website: ${stateResource.contact}</i>\n`;
-  });
+  const data = [`\n\n\u{1F9EA} COVID-19 Testing Labs in ${stateResources[0].state}, found ${stateResources.length} results\n`];
+  data[0] += `\n<b>${stateResources[0].nameoftheorganisation}, ${stateResources[0].city}\n`
```

Comment:
Why can't we merge this with the `for loop`?

Reply:
If we have to merge it in for loop, we will have to use `data[i]+=....` which will work good for the 1st element but for rest using `data[i]+=...` in very first line was producing undefined on top like this.
![gur2](https://user-images.githubusercontent.com/57032785/85929304-4f84cb80-b8d1-11ea-84ec-39eb0e0b67c3.png)

Reply:
I am changing it to include in for loop with if else which will work similarly. 👍